h/static/scripts/test/build-thread-test.js

```diff
@@ -262,6 +263,44 @@ describe('build-thread', function () {
         }]);
       });
     });
+
+    context('thread filtering', function () {
```

Comment:
AIUI when you have a set of tests _about_ a particular function/class etc., those are `describe()` blocks. When you have a set of tests that apply _when_ certain a certain condition is met, those are `context()` blocks.